src/main/java/codewars/tasks/java/LeapYears.java

```diff
@@ -0,0 +1,19 @@
+package codewars.tasks.java;
+
+public class LeapYears {
+
+    public boolean isYearLeap(int year){
+        boolean isYearLeap = false;
+
+        if(year % 100 == 0){
+            isYearLeap = false;
+        } else if (year % 400 == 0){
+            isYearLeap = true;
+        } else if (year % 4 == 0){
+            isYearLeap = true;
+        }
```

Comment:
Try to simplify `else if (year % 400 == 0 || year % 4 == 0) { isYearLeap = true; }`